generate-release-cycle.py

```diff
@@ -0,0 +1,136 @@
+"""
+Read in a JSON and generate two CSVs and a Mermaid file
+"""
+from __future__ import annotations
+
+import csv
+import datetime as dt
+import json
+
+MERMAID_HEADER = """
+gantt
+    dateFormat  YYYY-MM-DD
+    title       Python release cycle
+    axisFormat  %Y
+""".lstrip()
+
+MERMAID_SECTION = """
+    section Python {version}
+    {release_status}               :{mermaid_status}   python{version}, {release_date},{eol}
+"""  # noqa: E501
+
+MERMAID_STATUS_MAPPING = {
+    "feature": "",
+    "bugfix": "active,",
+    "security": "done,",
+    "end-of-life": "crit,",
+}
+
+CSV_HEADER = (
+    "Branch",
+    "Schedule",
+    "Status",
+    "First release",
+    "End of life",
+    "Release manager",
+)
```

Comment:
These are duplicated down below; my suggested refactoring https://github.com/CAM-Gerlach/devguide/commit/c2cbf6b0326d6b63c3a35934529c8247f83d5785 fixes this